generator/src/test/java/com/scottlogic/deg/generator/smoke_tests/ExampleProfilesTests.java

```diff
@@ -40,7 +40,7 @@
                 GenerationConfig.DataGenerationType.INTERESTING,
                 GenerationConfig.TreeWalkerType.CARTESIAN_PRODUCT,
                 GenerationConfig.CombinationStrategyType.PINNING));
-            final Profile profile = new ProfileReader().read(profileFile.toPath());
+            final Profile profile = new ProfileReader().read(profileFile.toPath(), GenerationConfig.Constants.DEFAULT_PROFILE_VALIDATION );
```

Comment:
Should validation be explicitly enabled/disabled here?
Maybe the smoke tests should be employing the validator, thats fine - but maybe it should be explicit.

Reply:
changed.